python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -57,6 +57,7 @@
 
 
 class RasterizeAlgorithm(QgisAlgorithm):
+
```

Comment:
Yes you are right. I can't find the reference for *before*. But AFAIK, a lot of tools (http://landscape.io http://sideci.com, pep257 package) are checking this.

Nevermind as you said. It might be good to look for some PEP257 tools on qgis later.